src/person_operations.rb

```diff
@@ -0,0 +1,39 @@
+require './src/person'
+require './src/modules/list'
+
+class PersonOperations
+  def initialize(person)
+    @people = person
+  end
+
+  include List
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      return
+    end
+
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    if person_type == '1'
+      print 'Has parent permission? [Y/N]: '
+      parent_permission = gets.chomp
+      parent_permission = parent_permission.downcase == 'y'
+      student = Student.new(name, parent_permission, age)
```

Comment:
Great job team 🤝

- [ ] According to the Single Responsibility Rule, `all requests for input and input validation should've been placed in the main.rb file`. Kindly check out this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more guidance.

Reply:
Input and validation request placed on the main.rb file